lib/Gitlab/Api/Groups.php

```diff
@@ -119,6 +119,20 @@ public function members($id, array $parameters = [])
         return $this->get('groups/'.$this->encodePath($id).'/members', $resolver->resolve($parameters));
     }
 
+    /**
+     * @param int   $group_id
+     * @param int   $user_id
+     *
+     * @return mixed
+     */
+    public function member($group_id, $user_id)
+    {
+        $resolver = $this->createOptionsResolver();
+        $resolver->setDefined('query');
+
```

Comment:
$resolver is unused, too much copy-pasting and not testing your added functionality?

I personally would add @see annotation and link to api doc, so could verify what parameters are supported